app.css

```diff
@@ -24,6 +26,12 @@ a{
 :root{
     --primary: blueviolet;
     --white: #fff;
+    --secondary: #155d9c;
+    --success: #198754;
+    --danger: #DC3545;
+    --warning: #FFC107;
+    --info: #0DCAF0;
+    --dark: #212529;
```

Comment:
You have already defined it in components/button.css which you have imported into this file. So these lines are redundant and you can remove them

Reply:
Yeah